add_up.rb

```diff
@@ -0,0 +1,14 @@
+def add_up(i)
+    for a in 1..i do
+        a+=a
+    end
+number_full = a + i
```

Comment:
We don't need line 5. For line 6, we display the sum like - `puts sum` instead of `puts number_full`